### DISCUSSION TITLE
Knowledge-Base Release v0.2.0

### DISCUSSION THREAD

**Discussion by zivy** [IBEXImagingCommunity/ibex_imaging_knowledge_base#197] [Announcements]
We are proud to announce the second authoritative release of the IBEX Knowledge-Base.

Download version v0.2.0 from Zenodo [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.13122300.svg)](https://doi.org/10.5281/zenodo.13122300). For the detailed list of changes, additions and most importantly, first time contributors, please see the [release notes](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/releases/tag/v0.2.0).

We thank all contributors, old and new, for sharing their Knowledge with the community!

p.s. Contributed to the Knowledge-Base? List the dataset in your biosketch as part of your contributions to science.

**Comment by radtkea** [comment]
Hooray!!! So much progress!
